lib/active_model/serializer/configuration.rb

```diff
@@ -22,6 +22,8 @@ def config.array_serializer
         config.default_includes = '*'
         config.adapter = :attributes
         config.jsonapi_resource_type = :plural
+        config.jsonapi_namespace_separator = '--'.freeze
+        config.jsonapi_type_transform = :underscore
```

Comment:
Yup... This is why it needs a thoughtful review :)

B mobile phone

> On Jun 9, 2016, at 3:54 AM, L. Preston Sego III <notifications@github.com> wrote:
> 
> In lib/active_model/serializer/configuration.rb:
> 
> > @@ -22,6 +22,8 @@ def config.array_serializer
> >          config.default_includes = '*'
> >          config.adapter = :attributes
> >          config.jsonapi_resource_type = :plural
> > +        config.jsonapi_namespace_separator = '--'.freeze
> > +        config.jsonapi_type_transform = :underscore
> type transform is underscore? doesn't this go against our other transform?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>